tap_workday_raas/discover.py

```diff
@@ -86,7 +86,7 @@ def generate_schema_for_report(xsd):
 
             max_occurs = elem.attrib.get("maxOccurs")
             if max_occurs not in (None, "unbounded"):
-                raise Exception("Found unexpected value for maxOccurs attribute.")
+                raise Exception("Found unexpected value for maxOccurs attribute: '{}'".format(maxOccurs))
```

Comment:
Should be `max_occurs`

Reply:
And there should be another spot with this I think?

Reply:
Done.